.github/workflows/spelling-check-bot.yml

```
@@ -0,0 +1,49 @@
+name: Create an issue to report typos
+
+on:
+  schedule:
+    - cron: "0 0 * * mon"
+  workflow_dispatch:
+
+jobs:
+  sync:
+    if: github.repository == 'mdn/content'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: mdn/content
+          ref: main
+          path: content
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+          cache: yarn
+
+      - name: Run scripts
+        run: |
+          npm install
+          echo Running spelling check...
+          output=$(npx cspell --no-progress --config .vscode/cspell.json "content/**/*.md" || exit 0)
+          echo "$output"
+          echo "OUTPUT<<EOF" >> $GITHUB_ENV
+          echo "$output" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
+
+      - name: Create an issue
+        if: env.OUTPUT != ''
+        uses: dacbd/create-issue-action@main
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          title: Weekly spelling check
+          body: |
+            ## Found typos and unknown words:
+            ${{ env.OUTPUT }}
```

Does this make sense to put it in a code block?

```suggestion
            Typos and unknown words:
            
            ```
            ${{ env.OUTPUT }}
            ```
```